language/German/strings.po

```diff
@@ -721,23 +724,21 @@ msgstr "Zurück zum vorherigen Menü"
 msgid "File Manager"
 msgstr "Dateimanager"
 
-#: src/frontend/mame/ui/filemngr.cpp:140
-#: src/frontend/mame/ui/confswitch.cpp:119
-#: src/frontend/mame/ui/inputmap.cpp:580
-#: src/frontend/mame/ui/inputtoggle.cpp:75
+#: src/frontend/mame/ui/filemngr.cpp:140 src/frontend/mame/ui/confswitch.cpp:119
+#: src/frontend/mame/ui/inputmap.cpp:580 src/frontend/mame/ui/inputtoggle.cpp:75
```

Comment:
Can you please back out this one as well?

Reply:
Fixed